File: tempCodeRunnerFile.py
"""

https://www.acmicpc.net/problem/2231
"""


import math
import sys


def input(): return sys.stdin.readline().rstrip()


X, Y = map(int, input().split())

Z = int((Y/X)*100)

# Z+1 이 목표

dZ = (((Y+1)/(X+1))*100) - (((Y)/(X))*100)
#print("->", X, Y, Z, dZ)
a = dZ

if Z != 99 and round(a, 9) != 0:
    # 가능성
    print(math.ceil(1/a))
else:
    print(-1)

File: env_node/src/tempCodeRunnerFile.js

let ans = 0;
for (let i = 0; i <= 7; i++) {
  const row = input().split("");
  for (let j = 0; j <= 7; j++) {
    if ((i + j) % 2 === 0 && row[j] === "F") {
      console.log(i, j, row[j], row);
      ans += 1;
    }
  }
}
console.log(ans);
